test/e2e/utils/cli.go

```diff
@@ -34,6 +34,10 @@ func Tanzu(input io.Reader, args ...string) (string, error) {
 	return cliRunner("tanzu", input, args...)
 }
 
+func Aws(input io.Reader, args ...string) (string, error) {
```

Comment:
Given that we don't have any usage of `input` from the external callers, can we get rid of it for now? And do `cliRunner("aws", nil, args...)`

Reply:
thanks @karuppiah7890 for looking into this PR.
Just want to clear one thing you are saying to invoke the complete script at   https://github.com/vmware-tanzu/community-edition/blob/main/hack/ensure-dependencies.sh and invoke it like this
`"${TCE_REPO_PATH}/hack/ensure-dependencies.sh"` ?
If yes then the other tools which this code is installing will also get install right but yes max tools we required in any-case like Jq, kubectl, docker etc just want to confirm if i am getting you wrong.

Reply:
Thanks Karuppiah for looking into this PR.
Make sense I will do the following changes.